src/mixins/filing-template-mixin.ts

```diff
@@ -252,6 +251,15 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
       }
     }
 
+    // Apply Name Request data if applicable
+    if (this.getNameRequest.nrNumber) {
+      filing.alteration.nameRequest = {
+        legalType: this.getNameRequest.legalType,
+        legalName: this.getNameRequest.legalName,
+        nrNumber: this.getNameRequest.nrNumber
+      }
+    }
+
```

Comment:
I'm guessing this is part of a different ticket :)

Reply:
I'm not sure this is captured in another ticket, just that i noticed we were passing a lot of unwanted properties to the api in the event of company name change. This extra properties were causing the api to error.

Figured i'd fix it proactively before i saw a bug ticket 😁

Reply:
Cool! got it

Reply:
Sounds good. Please call it out in the PR description in case of future post-mortem 😁

Reply:
Done!